layers/+filetree/treemacs/README.org

```diff
@@ -79,7 +80,20 @@ layer variable =treemacs-use-collapsed-directories= to a positive number.
     (treemacs :variables treemacs-use-collapsed-directories 3)))
 #+END_SRC
 
-Default value is 3 (or 0 when python is not installed).
+Default is 3 (or 0 when python is not installed).
+
+** Locking width
+To have the width of the treemacs window locked by default, meaning not manually
```

Comment:
Sorry for nitpicking, but I don't understand this description. If you wanted to copy the description style of `treemacs-use-filewatch-mode`, then you missed the part where it says what to do in order to get the desired behaviour. Like this:

> To have the width of the treemacs window locked by default (meaning the inability to resize treemacs window) set the layer variable `treemacs-lock-width` to non-nil.

Reply:
Rewriting :thumbsup::sparkles: